### DISCUSSION TITLE
Battery powered?

### DISCUSSION THREAD

**Discussion by jacobdonenfeld** [HASwitchPlate/HASPone#208] [Ideas]
Anybody have luck powering this off of a battery? Would be a fun DIY mod to install a micro-b port on the front to charge this. Would make for some easier installs for me.

**Comment by PhaseAngle** [comment]
I'm in the process of building one with a LoLin so I can power from a usb plug pack located in the ceiling.
This keeps me within the tight Australian electrical rules
